### DISCUSSION TITLE
Feature request: Advertising for less than one second or a specific number of broadcasts.

### DISCUSSION THREAD

**Discussion by sivar2311** [h2zero/NimBLE-Arduino#388] [Q&A] [answered]
```NimBLEAdvertising::start()``` uses the duration in seconds.

Is it possible to have a `NimBLEAdvertising::start_ms()` function that accepts the duration in milliseconds? 
Alternatively, a function that accepts the number of advertisements as a parameter would be interesting (`NimBLEAdvertising::start_count()`)

_Note:_
I don't yet know the BLE internals very well and I don't understand the full meaning of the functions
`setMinInterval`
`setMaxInterval`
`setMinPrefered`
`setMaxPrefered`

**Comment by h2zero** [comment, marked as answer]
I'm planning to change all of the functions that accept a time parameter to use milliseconds instead of seconds. This is a breaking change though so it will be in a later release.
